forge/src/multi_runner.rs

```diff
@@ -802,8 +802,8 @@ mod tests {
                             None,
                             Some(vec![
                                 "constructor".into(),
-                                "testMisc, 0x0000000000000000000000000000000000000001".into(),
-                                "testMisc, 42".into(),
+                                "testMisc 0x0000000000000000000000000000000000000001".into(),
```

Comment:
Note here that the existing console.log output changes with this patch.. Even if `%s` isn't used. The new behavior is more inline with hardhat's so I don't think it'll be a huge issue for users.